specs/wp-signup-spec.js

```diff
@@ -1399,4 +1400,87 @@ test.describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function()
+		const expectedDomainName = `${ blogName }.art.blog`;
+
+		test.it( 'Ensure we are not logged in as anyone', async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		test.it( 'Can enter the subdomains flow and select design type', async function() {
+			await driver.get( config.get( 'calypsoBaseURL' ) + '/start/subdomain/?vertical=a8c.1' );
+			const designTypePage = await DesignTypePage.Expect( driver );
+			await designTypePage.setABTestControlGroupsInLocalStorage( { flow: 'subdomain' } );
+			return await designTypePage.selectFirstDesignType();
+		} );
+
+		test.it(
+			'Can see the choose a theme page as the starting page, and select the first theme',
+			async function() {
+				const chooseAThemePage = await ChooseAThemePage.Expect( driver );
+				return await chooseAThemePage.selectFirstTheme();
+			}
+		);
+
+		test.it(
+			'Can then see the domains page, and Can search for a blog name, can see and select a free .art.blog address in the results',
+			async function() {
+				const findADomainComponent = await FindADomainComponent.Expect( driver );
+				await findADomainComponent.searchForBlogNameAndWaitForResults( blogName );
+				await findADomainComponent.checkAndRetryForFreeBlogAddresses(
+					expectedDomainName,
+					blogName
+				);
+				let actualAddress = await findADomainComponent.freeBlogAddress();
+				assert(
+					expectedDomainName.indexOf( actualAddress ) > -1,
+					`The displayed blog address: '${ actualAddress }' was not the expected addresses: '${ expectedDomainName }'`
+				);
+				return await findADomainComponent.selectFreeAddress();
+			}
+		);
+
+		test.it( 'Can then see the plans page and pick the free plan', async function() {
+			const pickAPlanPage = await PickAPlanPage.Expect( driver );
+			return await pickAPlanPage.selectFreePlan();
+		} );
+
+		test.it( 'Can then enter account details and continue', async function() {
+			const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
+			const createYourAccountPage = await CreateYourAccountPage.Expect( driver );
+			return await createYourAccountPage.enterAccountDetailsAndSubmit(
+				emailAddress,
+				blogName,
+				passwordForTestAccounts
+			);
+		} );
+
+		test.it(
+			"Can then see the sign up processing page -  will finish and show a 'Continue' button which is clicked",
+			async function() {
+				await SignupProcessingPage.hideFloatiesinIE11( driver );
+				const signupProcessingPage = await SignupProcessingPage.Expect( driver );
+				await signupProcessingPage.waitForContinueButtonToBeEnabled();
+				return await signupProcessingPage.continueAlong();
+			}
+		);
+
+		test.it(
+			'We are on the view blog page, can see trampoline, our URL and title',
+			async function() {
+				const viewBlogPage = await ViewBlogPage.Expect( driver );
+				await viewBlogPage.waitForTrampolineWelcomeMessage();
+				let displayed = await viewBlogPage.isTrampolineWelcomeDisplayed();
+				return assert.strictEqual(
```

Comment:
Could you assert we're seeing the correct .blog URL as per the title of this block?